docs/source/topics/build-examples.rst

```diff
@@ -26,24 +26,53 @@ basic usage scenarios of |short_name| with SYCL*. Go to
 
 .. note::
 
-   All content starting with a # below is considered a comment and
+   All content below that starts with ``#`` is considered a comment and
    should not be run with the code.
 
 1. Set up the |short_name| environment:
 
-  **Linux\*:**
+  .. tabs::
 
-    .. substitution-prompt:: bash
+    .. group-tab:: Linux\*
 
-      # Run script to setup CPATH, LIBRARY_PATH and LD_LIBRARY_PATH for |short_name|
-      source ./env/vars.sh
+      On Linux, there are two possible ways to set up |short_name| environment:
+      via ``vars.sh`` script or via ``modulefiles``.
 
-  **Windows\*:**
+      * Setting up |short_name| environment via ``vars.sh`` script
 
-    .. substitution-prompt:: bash
+        Run ``vars.sh`` script to set up ``CPATH``, ``LIBRARY_PATH``, and ``LD_LIBRARY_PATH`` for |short_name|:
```

Comment:
Might be it's worth saying that this will set required environment for library such as ``CPATH``, ``LIBRARY_PATH``, and ``LD_LIBRARY_PATH``.  Because there are might be additional things set by this script

It's ok to skip this for now

Reply:
changed the wording